src/fu-util.c

```diff
@@ -410,12 +420,11 @@ fu_util_perhaps_show_unreported (FuUtilPrivate *priv, GError **error)
 				      "Firmware updates were applied successfully",
 				      unreported_success));
 	}
-	g_printerr ("%s\n > fwupdmgr report-history\n",
-		    /* TRANSLATORS: what the user has to do, command follows */
-		    _("To share useful information with the developers and "
-		      "clear this message use the following command:"));
 	g_printerr ("\n******************\n");
-	return TRUE;
+	g_printerr ("%s [Y|n]: ", _("Share this information now?"));
```

Comment:
Same as other area, guard the display of this statement based upon `priv->assume_yes`, `priv->assume_no`

Reply:
Fixed.